airflow/providers/amazon/aws/example_dags/example_salesforce_to_s3.py

```diff
@@ -15,60 +15,35 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# Ignore missing args provided by default_args
-# type: ignore[call-arg]
-
 """
 This is a basic example DAG for using `SalesforceToS3Operator` to retrieve Salesforce customer
```

Comment:
```suggestion
This is a basic example DAG for using `SalesforceToS3Operator` to retrieve Salesforce account
```
Since the query is changing.

Reply:
Nice one. Fixed